bin/tachyon-mount.sh

```diff
@@ -137,7 +137,7 @@ function mount_ramfs_linux() {
   fi
 
   mem_size_to_bytes
-  FREE_MEM=`free -b | grep "^Mem" | awk '{print $2}'`
+  FREE_MEM=$(($(cat /proc/meminfo | awk 'NR==1{print $2}') * 1024))
```

Comment:
This is actually total memory and not free memory right? I think we always wanted total memory, perhaps change the variable name?